content/doc/administration/requirements/java.adoc

```diff
@@ -9,11 +9,12 @@ There are separate run and job execution requirements for Jenkins installations.
 
 Modern Jenkins versions have the following Java requirements:
 
-* Java 8 is the **ONLY** currently supported runtime environment, both 32-bit and 64-bit versions are supported
+* Java 8 runtime environments, both 32-bit and 64-bit versions are supported
+* Since Jenkins `2.164` footnote:[since `2.155` but it was made easier in `2.164`], Java 11 runtime environments are supported
```

Comment:
```suggestion
* Java 11 is supported starting from 2.164
```

We do not formally support Java 11 for previous Jenkins versions, probably we should not overcomplicate it here

Reply:
Given this was already hinted in a footnote above, I tend to agree this may be a unnecessary repetition about the pre-2.164 story.